src/meanshift.jl

```diff
@@ -0,0 +1,96 @@
+"""
+```
+segments                = meanshift(img, spatial_radius, range_radius, [iter], [eps])
```

Comment:
Use `; iter=50, eps=0.01)` to clearly indicate that these are keyword arguments and what their default values are. (Before I looked at the code, I thought these were positional arguments and was wondering how you guessed which was which.)